holdings_maintenance/pds4/pds4infoshelf.py

```
@@ -0,0 +1,931 @@
+sys.path.insert(0, str(REPO_ROOT))
+
+from holdings_maintenance.pds4 import pds4checksums
+
+# Holds log file directories temporarily, used by move_old_info()
+LOGDIRS = []
+
+LOGNAME = 'pds.validation.fileinfo'
+LOGROOT_ENV = 'PDS_LOG_ROOT'
+
+PREVIEW_EXTS = set(['.jpg', '.png', '.gif', '.tif', '.tiff',
+                    '.jpeg', '.jpeg_small'])
```

Ruff (0.14.0)

Unnecessary list literal (rewrite as a set literal)

Rewrite as a set literal

(C405)